### DISCUSSION TITLE
Making Changes in Codebase

### DISCUSSION THREAD

**Discussion by itsaboutcode** [crater-invoice-inc/crater#784] [General]
I setup the code using docker with the help of following guide - https://docs.craterapp.com/installation.html#docker-installation

Now I am making changes in js files but they are not reflecting. Any ideas?

Thanks

**Comment by mohitpanjwani** [comment]
Hey @itsaboutcode, I think I might be replying a little too late. 

but you need to compile the JS files using `yarn build`(for production) or `yarn dev`(If you're developing locally). 

Make sure to set APP_ENV to `local` on the .env file if you're running `yarn dev` otherwise it won't work.

**Reply by maha-oueghlani** [reply]
@mohitpanjwani  Hello I'm developping on local but when I run `npm run dev` it can't do changes
